Rakefile

```diff
@@ -0,0 +1,17 @@
+require 'bundler/gem_tasks'
+
+require 'rspec/core/rake_task'
+RSpec::Core::RakeTask.new(:spec)
+
+require 'rubocop/rake_task'
+RuboCop::RakeTask.new(:rubocop)
+
+desc 'run specs and rubocop'
+task ci: [:spec, :rubocop]
+
+task default: :ci
+
+# the next 3 lines are useful for manual testing of THIS gem
```

Comment:
Is there any harm in uncommenting these? It seems pretty useful if you're trying to develop on the gem.